### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by abachtold** [HospitalRun/hospitalrun-frontend#2889] [Q&A]
Is there a page or list of the translations for the hospitalrun? I need de Portuguese Brazil translation, but I can't find if that's available.

**Comment by xiaobaolan** [comment]
So I believe that this issue was treated earlier by some Brazilian users: Here is the link to the PR below.

https://github.com/HospitalRun/hospitalrun-frontend/issues/2769

I am not sure if this helps you but let me know.
